SYCL/FilterSelector/reuse.cpp

```diff
@@ -15,6 +15,8 @@
 #include <CL/sycl.hpp>
 
 using namespace cl::sycl;
+// change to 'using namespace cl::sycl::oneapi' after PR intel/llvm:4014 is
```

Comment:
Is it a TODO or description of what's done?

Reply:
Sorry, missed `TODO: `, fixed